objectnode/auth_signature_v2.go

```diff
@@ -1,6 +1,6 @@
 /*
+ * Copyright 2018 The ChubaoFS Authors.
  * MinIO Cloud Storage, (C) 2018 MinIO, Inc.
- * Modified work copyright 2020 The ChubaoFS Authors.
  *
```

Comment:
MinIO Cloud Storage, (C) 2018 MinIO, Inc.
Modifications copyright 2020 The ChubaoFS Authors.